docs/src/pages/customization/components/ClassNames.js

```diff
@@ -1,24 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import { withStyles } from '@material-ui/core/styles';
+import { createStyles, makeStyles } from '@material-ui/core/styles';
 import Button from '@material-ui/core/Button';
 
 // We can inject some CSS into the DOM.
-const styles = {
-  root: {
-    background: 'linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)',
-    borderRadius: 3,
-    border: 0,
-    color: 'white',
-    height: 48,
-    padding: '0 30px',
-    boxShadow: '0 3px 5px 2px rgba(255, 105, 135, .3)',
-  },
-};
+const useStyles = makeStyles(() =>
```

Comment:
This should work without the callback and `createStyles`. Plain object should work.

Reply:
Oh sorry, I will try resolve this